pytest_sentry.py

```diff
@@ -4,12 +4,13 @@
 import pytest
 
 import wrapt
+
 import sentry_sdk
+from sentry_sdk.opentelemetry.scope import PotelScope as Scope, use_scope
```

Comment:
please import the toplevel scope, the switch should be internal to the SDK
```
from sentry_sdk import Scope, use_scope
```

Reply:
done